### ISSUE TITLE
Is it possible to add build phase from pbxproj command

### ISSUE THREAD

**Issue by jkmathew** [kronenthaler/mod-pbxproj#163]
Is it possible to add build phase (especially runs scrip) from pbxproj command
I am expecting something like,
`pbxproj --add-run-script "echo 'its here'"`

**Comment by kronenthaler** [comment]
Thanks for the feedback, it can be included in a future release. I can't ensure when that will happen though.

**Comment by kumarpatel** [comment]
isn't this possible via `flags` CLI option?

**Comment by kronenthaler** [comment]
@kumarpatel no, flags modify the configuration settings of the target. The run scripts are added on a build phase of the target. 
This is marked as an enhancement that will be implemented in the future. Or feel free to contribute it with a PR.
